### ISSUE TITLE
Pre Processsing error at CRX file

### ISSUE THREAD

**Issue by earth-obs** [goGPS-Project/goGPS_MATLAB#59]
goGPS doesn't create any file in C:\...\satellite\CRX directory. And I'm getting this error message. How to do I? Is it possible to deactivate pre-processing because of my data not regularly?

  **   Pre-processing on receiver 1: log12040207a
  --------------------------------------------------------------------

   **   Removing observations marked as bad in Bernese .CRX file
   **   Reading CRX data
 [ WW ] Warning: The required (updated) CRX files were not found in C:\goGPS_MATLAB-goGPS_1.0_beta\data\satellite\CRX directory.
                 

   **   Removing observations for which no ephemerid or clock is present
Dot indexing is not supported for variables of this type.

Error in  -  (line 1695)
            [unix_time2, unix_time_f2] = gt_2.getUnixTime();

Error in Receiver_Work_Space/remBad (line 1030)
            coord_ref_time_diff = first_epoch - this.sat.cs.time_ref_coord;

Error in Receiver_Work_Space/preProcessing (line 7702)
                this.remBad();

Error in Command_Interpreter/runPrePro (line 882)
                        rec(r).work.preProcessing();

Error in Command_Interpreter/exec (line 713)
                            this.runPrePro(rec, tok(2:end));

Error in Core/exec (line 883)
            this.cmd.exec(this.rec, cmd_list, level);

Error in Core/go (line 845)
                            this.exec(cmd_list(execution_block == eb), sss_level(execution_block == eb));

Error in goGPS (line 137)
            core.go(); % execute all

**Comment by clicat** [comment]
Thanks for the report, I'll try to reproduce the error and correct it in the morning! 
The pre-processing perform multiple tasks that are vital for PPP or NET processing (apply multiple corrections, compute tropospheric delay, synchronize receiver etc). You can't deactivate it. But you can create a CRX folder and download the files manually from the aiub ftp.

**Comment by earth-obs** [comment]
But my data isn't IGS point, and can you give me a link, please?

**Comment by clicat** [comment]
I reviewed the download functions, I was not really able to reproduce the bug, but I modified and corrected some procedures. Give the new code a try!
The CRX file contains a list of satellite problems, and they can be found here: ftp://ftp.aiub.unibe.ch/BSWUSER52/GEN/ one file per year
e.g. ftp://ftp.aiub.unibe.ch/BSWUSER52/GEN/SAT_2019.CRX

However, I do not understand what's the meaning of the "IGS" sentence, why an IGS point should be special?

If you find the bug again send me the complete log, maybe I can understand a little better if the problem was generated due to some previous misconfiguration.

And thanks again for trying to use goGPS, I know that with no documentation (not yet ready) it can be hard!
